packages/router/src/router.tsx

```diff
@@ -66,13 +65,12 @@ const InternalRoute: React.VFC<InternalRouteProps> = ({
   name,
   redirect,
   notfound,
-  whileLoading = () => null,
+  // @ts-expect-error - This prop is picked up by <Set>
+  whileLoading, // eslint-disable-line
```

Comment:
Super annoying that ESLint doesn't understand `ts-` compiler silencers.

Reply:
I know!